.github/workflows/ocs-operator-ci.yaml

```diff
@@ -69,7 +73,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: ["1.17", "1.18"]
+        go: ["1.18", "1.19"]
         make-target: ["verify-deps", "verify-generated", "verify-latest-csv", "verify-operator-bundle", "verify-latest-deploy-yaml"]
```

Comment:
Do we need this make-target matrix any more?

Reply:
Good catch!

Reply:
Removed, and also fixed some of the jobs that had wrong targets :joy: